src/openrct2/ride/Ride.h

```diff
@@ -639,48 +639,48 @@ enum
     RIDE_STATUS_COUNT,
 };
 
-enum : uint8_t
+enum class RideMode : uint8_t
 {
-    RIDE_MODE_NORMAL,
-    RIDE_MODE_CONTINUOUS_CIRCUIT,
-    RIDE_MODE_REVERSE_INCLINE_LAUNCHED_SHUTTLE,
-    RIDE_MODE_POWERED_LAUNCH_PASSTROUGH, // RCT2 style, pass through station
-    RIDE_MODE_SHUTTLE,
-    RIDE_MODE_BOAT_HIRE,
-    RIDE_MODE_UPWARD_LAUNCH,
-    RIDE_MODE_ROTATING_LIFT,
-    RIDE_MODE_STATION_TO_STATION,
-    RIDE_MODE_SINGLE_RIDE_PER_ADMISSION,
-    RIDE_MODE_UNLIMITED_RIDES_PER_ADMISSION = 10,
-    RIDE_MODE_MAZE,
-    RIDE_MODE_RACE,
-    RIDE_MODE_DODGEMS,
-    RIDE_MODE_SWING,
-    RIDE_MODE_SHOP_STALL,
-    RIDE_MODE_ROTATION,
-    RIDE_MODE_FORWARD_ROTATION,
-    RIDE_MODE_BACKWARD_ROTATION,
-    RIDE_MODE_FILM_AVENGING_AVIATORS,
-    RIDE_MODE_3D_FILM_MOUSE_TAILS = 20,
-    RIDE_MODE_SPACE_RINGS,
-    RIDE_MODE_BEGINNERS,
-    RIDE_MODE_LIM_POWERED_LAUNCH,
-    RIDE_MODE_FILM_THRILL_RIDERS,
-    RIDE_MODE_3D_FILM_STORM_CHASERS,
-    RIDE_MODE_3D_FILM_SPACE_RAIDERS,
-    RIDE_MODE_INTENSE,
-    RIDE_MODE_BERSERK,
-    RIDE_MODE_HAUNTED_HOUSE,
-    RIDE_MODE_CIRCUS_SHOW = 30,
-    RIDE_MODE_DOWNWARD_LAUNCH,
-    RIDE_MODE_CROOKED_HOUSE,
-    RIDE_MODE_FREEFALL_DROP,
-    RIDE_MODE_CONTINUOUS_CIRCUIT_BLOCK_SECTIONED,
-    RIDE_MODE_POWERED_LAUNCH, // RCT1 style, don't pass through station
-    RIDE_MODE_POWERED_LAUNCH_BLOCK_SECTIONED,
-
-    RIDE_MODE_COUNT,
-    RIDE_MODE_NULL = 255,
+    NORMAL,
```

Comment:
I know we haven't done it for every enum (yet) but each element should be TitleCase